tasks/config.yml

```diff
@@ -7,13 +24,19 @@
     all_files:
       - type: link
         files: "{{ systemd_networkd_link | dict2items }}"
+        dest_folder: "{{ systemd_networkd_directory }}"
       - type: netdev
         files: "{{ systemd_networkd_netdev | dict2items }}"
+        dest_folder: "{{ systemd_networkd_directory }}"
       - type: network
         files: "{{ systemd_networkd_network | dict2items }}"
+        dest_folder: "{{ systemd_networkd_directory }}"
+      - type: conf
+        files: "{{ systemd_networkd_conf | dict2items }}"
+        dest_folder: "{{ systemd_networkd_conf_directory }}"
```

Comment:
Again, I'm really confused about that. Can you point me out to some docs about what would go in `/etc/systemd/networkd.conf.d`? I can't find anything about it, and I don't understand why you would need `systemd_networkd_directory` and `systemd_networkd_conf_directory`.

Reply:
It's a matter of personal preference.
On this [documentation](https://www.freedesktop.org/software/systemd/man/latest/networkd.conf.html#), you can see that configuration files can be written in three places.

- `/etc/systemd/networkd.conf`
- `/etc/systemd/networkd.conf.d/*.conf`
- `/usr/lib/systemd/networkd.conf.d/*.conf`

I choose to put it in `conf.d` folder as an habit, not knowing if I would implement multiple files or not, it appears that I didn't implemented multiple conf files though.  

In the current state of the role, we can choose `/etc/systemd/networkd.conf` as the `.conf` destination with no downside.
I can implement this change if you have a preference, removing the second default variable & the `dest_folder` from the main loop.

Reply:
That totally makes sense, I was confused that you used `/etc/systemd/networkd.conf.d` as an alternative to `/etc/systemd/network`. Sorry for the confusion!

Reply:
No problem !
But you made a valid point, it add confusion to the role with no functionality improvement, so it make sense to simply use `/etc/systemd/network.conf`.